testsuite/shared/src/main/java/org/jboss/as/test/integration/domain/management/util/DomainTestUtils.java

```diff
@@ -203,7 +204,32 @@ public static void waitUntilState(final ModelControllerClient client, final Stri
      * @throws IOException
      */
     public static void waitUntilState(final ModelControllerClient client, final PathAddress serverAddress, final String state) throws IOException {
-        waitUntilState(client, serverAddress, state, DEFAULT_TIMEOUT, TimeUnit.SECONDS);
+        waitUntilState(client, serverAddress, state, DEFAULT_TIMEOUT, TimeUnit.SECONDS, "status");
+    }
+
+    /**
+     * Wait until a server reached a given suspend state or fail if the timeout was reached.
+     *
+     * @param client        the controller client
+     * @param serverAddress the server address
+     * @param state         the required state
+     * @throws IOException
+     */
+    public static void waitUntilSuspendState(final ModelControllerClient client, final PathAddress serverAddress, final String state) throws IOException {
+        waitUntilState(client, serverAddress, state, DEFAULT_TIMEOUT, TimeUnit.SECONDS, "suspend-state");
+    }
+
+    /**
+     * Wait until a server reached a given suspend state or fail if the timeout was reached.
+     *
+     * @param client     the controller client
+     * @param hostName   the host name
+     * @param serverName the server name
+     * @param state      the required state
+     * @throws IOException
+     */
+    public static void waitUntilSuspendState(final ModelControllerClient client, final String hostName, final String serverName, final String state) throws IOException {
```

Comment:
This method looks unused anywhere. Is it intentionally kept in the MR or is it just a leftover?

Reply:
I used them along the feature development but they were not longer necesary. I've just removed them since are not used.